src/SDKs/_metadata/network_resource-manager.txt

```diff
@@ -1,11 +1,11 @@
-2018-04-26 20:59:33 UTC
+2018-05-15 23:11:28 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
-Branch:      master
-Commit:      b9721ae4c9f04914cc89260b8339dc33372f835b
+Branch:      Network-2018-05-01
```

Comment:
Apologize for missing this, we cannot accept PRs for sdks generated from a branch other than master. If your changes have already been merged in the Network May REST spec, you don't need this PR
Please talk to @MikhailTryakhov if you have any doubts